### ISSUE TITLE
Forgot to tag  v6.1.6?

### ISSUE THREAD

**Issue by jhhazelaar** [Really-Simple-Plugins/complianz-gdpr#364]
Hi,

Did you forget to tag v6.1.6? I see in de admin an update, but can't update the plugin with composer.

Thanks.

**Comment by rlankhorst** [comment]
Yes, I will do this first thing tomorrow. Thanks for reminding :)

**Comment by rlankhorst** [comment]
I'm sorry, free was not released yet. So what you see now is us getting ready for the release next week ( as you can see on Wordpress as well).

I thought I forgot it, but the release this week was of the premium plugin, not free. 

6.1.6 will be officially released next week.
